pkg/loadbalancer/reconciler/termination.go

```diff
@@ -105,6 +107,21 @@ func registerSocketTermination(p socketTerminationParams) error {
 		return nil
 	}
 
+	if err := sockets.InetDiagDestroyEnabled(p.Log, p.Config.LBSockTerminateAllProtos, true); err != nil {
+		if errors.Is(err, probes.ErrNotSupported) {
+			// The kernel doesn't support socket termination.
+			p.Log.Error("Forcefully terminating sockets connected to deleted service backends "+
+				"not supported by underlying kernel: see kube-proxy free guide for "+
+				"the required kernel configurations", logfields.Error, err)
```

Comment:
```suggestion
				"not supported by underlying kernel: see kube-proxy free guide for "+
				"the required kernel configurations", logfields.Error, err)
```

Just wondering - if the probing logic consistently returns which `CONFIG_*` it proped for (and failed), do we still need to point at the Cilium docs? Or is it enough to have the missing kconfig option called out in the error log?